src/editions/transitioning-an-existing-project-to-a-new-edition.md

```diff
@@ -119,4 +125,20 @@ As before, `cargo fix` may not be able to automatically update our code.
 If `cargo fix` can't fix something, it will print a warning to the console,
 and you'll have to fix it manually.
 
-Enjoy the new edition!
\ No newline at end of file
+Also as mentioned before, there are known bugs around the idiom lints which
+means they're not all ready for prime time yet. This means that you may get a
+scary-looking warning to report a bug to Cargo, which happens whenever a fix
+proposed by rustc actually caused code to stop compiling by accident. If you'd
+like `cargo fix` to make as much progress as possible, even if it causes code to
+stop compiling, you can execute:
+
+```console
+$ cargo fix --edition-idioms --broken-code
+```
+
+This will instruct `cargo fix` to apply changes even if they cause code to stop
```

Comment:
you already said "stop compiling" so the bit after "even" feels redundant either here or in the previous location.